app/src/app.py

```diff
@@ -21,6 +22,11 @@
 def create_app() -> APIFlask:
     app = APIFlask(__name__)
 
+    print("app => ", app)
+
+    if is_running_in_local():
+        setup_debug_adapter()
```

Comment:
We might want to put this `is_running_in_local` function in `local.py`.

https://github.com/navapbc/template-application-flask/blob/main/app/src/util/local.py